docs/src/less/ds4/index.less

```diff
@@ -189,7 +189,10 @@ footer[role="contentinfo"] {
 .demo {
     background: #fff;
     border: 1px solid #eee;
-    word-break: break-all;
+
+    figure.highlight .s {
+        word-break: break-all;
```

Comment:
What is the `s` class? Not sure I've seen that been used.

Reply:
Again, this deals with the code highlighting features in the docs. I think `.s` means "strings" but I've found that this fixes the overflow problems in the examples (without this certain strings blow out the mobile web view of our pages).

Reply:
Oh, sorry. I was thinking about `span.highlight`, not `figure.highlight`.

Reply:
No worries.